### ISSUE TITLE
JavaScript single-line comments begin with a double forward slash (//)

### ISSUE THREAD

**Issue by mikilion** [deanhume/html-minifier#16]
The inline JavaScript code with single-line comments begin with a double forward slash (//) causes to comment the remainder of javascript code on minimized html file; e.g.:

```
<script>
    var a; // this is a comment
    a = 3;
</script>
```

Minified:

```<script>var a; // this is a comment a = 3;</script>```

**Comment by deanhume** [comment]
Hi @mikilion - thanks for submitting this. I will look into it!

**Comment by Leon-O** [comment]
The solution is pretty easy: just remove the comment and then minimize the code.

**Comment by walterDurin** [comment]
Still an issue if no space between the double slash and the comment ie.
//this is a comment

**Comment by deanhume** [comment]
This has been fixed with the latest release of v1.5 :+1: 

https://github.com/deanhume/html-minifier/releases/tag/v1.5
